File: solutions/beecrowd/3170/3170.py
import sys

for line in sys.stdin:
    b = int(line)
    g = int(input())

    if g // 2 < b:
        print('Amelia tem todas bolinhas!')
    else:
        print(f'Faltam {(g // 2) - b} bolinha(s)')

File: solutions/beecrowd/3170/3170.clj
(ns main)

(defn main []
  (loop [line (read-line)]
    (when line
      (let [b (Integer/parseInt line)
            g (Integer/parseInt (read-line))]
        (if (< (int (/ g 2)) b)
          (println "Amelia tem todas bolinhas!")
          (printf "Faltam %d bolinha(s)%n" (- (int (/ g 2)) b))))
      (recur (read-line)))))

(main)
